packages/browser-wallet-api-helpers/src/wallet-api-types.ts

```diff
@@ -39,13 +39,14 @@ export enum EventType {
     ChainChanged = 'chainChanged',
 }
 
+export type SchemaTypeString = 'module' | 'parameter';
 export enum SchemaType {
     Module = 'module',
     Parameter = 'parameter',
 }
 
 export type SchemaWithContext = {
```

Comment:
```suggestion

export enum SchemaType {
    Module = 'module',
    Parameter = 'parameter',
}

type LaxStringEnumValue<E extends string> = `${E}`;

export type SchemaWithContext = {
    type: LaxStringEnumValue<SchemaType>;
    value: string;
};
```
then SchemaWithContext.type would be `'module' | 'parameter'` but without duplicating the keys.

Reply:
Hmm.. didn't think this would be possible without casting f.x. `SchemaType.Parameter` to a string, but it seems it works.. guess I remember wrong!

Reply:
Sorry, but I couldn't do a suggestion that would actually work.

Reply:
For once, I trusted the suggestion.. never again!

Should be fixed now though.